<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPageAttr</class>
 <widget class="QDialog" name="DialogPageAttr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>页属性</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>完整页名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdPageName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnPageNameSet">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>9</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>页类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>66</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>页描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdPageDesc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>62</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>91</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>图框</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdPageFrame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>87</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnPageFrameSet">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>86</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>276</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>标题栏属性</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>图号</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>页数</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>设计日期</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>审核</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>审核日期</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>工艺</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>校核</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>校核日期</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>标准化</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>标准化日期</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>批准</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>批准日期</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>标记</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>处数</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>签名</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>属性名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>数值</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>365</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>365</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdPageType">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>36</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
